src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -312,27 +383,17 @@ def dtype(self) -> DtypeObj:
         >>> idx.dtype
         dtype('int64')
         """
+        # TODO: SNOW-1458123 implement dtype
         WarningMessage.index_to_pandas_warning("dtype")
         return self.to_pandas().dtype
 
     @property
-    def T(self, *args: Any, **kwargs: Any) -> Index:
+    @index_not_implemented()
+    def shape(self) -> None:
         """
-        Return the transpose, which is by definition self.
-
-        Parameters
-        ----------
-        *args : Any
-            Optional positional arguments for compatibility with other T APIs.
```

Comment:
is this not supported any more?

Reply:
It just got moved further down (see Line 564)

Reply:
I see. Thanks!